src/three-domain-secure/component.jsx

```diff
@@ -134,6 +139,13 @@ export function getThreeDomainSecureComponent(): TDSComponent {
             };
           },
         },
+        locale: {
```

Comment:
The new `locale` prop aligns with what we pass here: https://github.com/paypal/paypal-checkout-components/blob/main/src/zoid/checkout/component.jsx#L190-L196